VimeoNetworking/VimeoNetworking/VimeoClient.swift

```diff
@@ -55,7 +55,10 @@ final public class VimeoClient
     }
     
         /// Dictionary containing URL parameters for a request
-    public typealias RequestParameters = [String: AnyObject]
+    public typealias RequestDictionary = [String: AnyObject]
```

Comment:
On that note (below) this name sounds kinda like a dictionary of requests. I think a name like RequestParameterDictionary would satisfy both concerns. A little longer but not bad.